src/_includes/layouts/base.njk

```diff
@@ -0,0 +1,47 @@
+<!doctype html>
+<html lang="{{ lang }}" dir="{{ config.languages[lang].dir }}">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width">
+
+        <title>{% if title %} {{ title }} | {% endif %} {{ site[lang].name }}</title>
+
+        {% include "partials/global/stylesheets.njk" %}
+
+        {% include "partials/global/meta.njk" %}
+        <link rel="alternate" type="application/atom+xml" href="{{ config.url }}{% if lang !== config.defaultLanguage %}/{{ lang }}{% endif %}/feed.xml" title="{{ translations[lang]["%s RSS feed"] | replace("%s", site[lang].name) }}">
+
+        {% include "partials/global/icons.njk" %}
+        <link rel="manifest" href="/manifest.webmanifest">
+
+        {% uioScripts %}
+        {% include "partials/global/scripts.njk" %}
+    </head>
+    {% block pageBody %}
```

Comment:
Could you add a comment to explain the home page uses this `pageBody` block to define its own body layout?